virus-interpreter/src/main/java/com/hartwig/hmftools/virusinterpreter/VirusInterpreterAlgo.java

```diff
@@ -104,6 +115,13 @@ boolean report(VirusBreakend virusBreakend, double expectedClonalCoverage, final
         return reported && virusQCStatus;
     }
 
+    @VisibleForTesting
+    boolean blacklist(int taxidGenus, int taxidSpecies)
+    {
+        return blacklistedTaxids.stream()
```

Comment:
You could also use `contains` perhaps with a simple list of integers? 

No super strong opinion but think it may be a bit more readable that way !(blacklisted.contains(genus) || blacklisted.contains(species))